### ISSUE TITLE
Since tuya-convert is becomming less and less affective...

### ISSUE THREAD

**Issue by dialupdisaster** [kueblc/mocktuyacloud#9]
Is there any hopes of this project picking back up?

**Comment by madbear2000** [comment]
This would be a nice work around for those of us that don't like our tuya switches calling out.
Also a docker container versions would be really appreciated.

**Comment by knifesk** [comment]
Yeah, I already purchased two smart switches that uses this new WA2 model from tuya.. they're not compatible with tasmota.. and I'M NOT USING their apps... I'll try to see what can I do with this project

**Comment by kueblc** [comment]
Hi all,

Thanks for your interest in this project.

My main goal with publishing this was to ensure my reverse engineering research would be available to others and not sitting in a directory going to waste. It is far from a polished product. While I would love to have it polished up and ready for the end user, I do not presently have the time to commit to this. Also, I no longer have any devices at home that run the original Tuya firmware.

I am happy to review and accept PRs, and would be thrilled to see the research adapted to other projects. If I am so lucky to find some time over the coming months, I will be happy to push forward on this once again, and will certainly let you all know.

As far as being an alternative to tuya-convert, it does not contain any firmware flashing related research that has not already been implemented in tuya-convert. As for sandboxing and controlling devices locally, codetheweb/tuyapi is a far more polished and ready to use solution.

**Comment by dachshund-digital** [comment]
Just one voice to add support, really really hope this becomes more mainstream as the various tuya flashing solutions are defeated.  Really resent tuya strategy forcing customers to only use their cloud.
